src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailService.java

```diff
@@ -21,6 +21,8 @@
 @Service
 public class EmailService implements IEmailService {
 
+    private static final String EXTERNAL_EMAIL_SUBJECT = "%s: Objection Application Submitted";
```

Comment:
I think you need to use the EMAIL_SUBJECT config value we defined and replace the placeholder, unless we think that approach is no longer needed.

Reply:
In my haste completely forgot this value was in configs, using that now